src/types/Car.ts

```diff
@@ -137,9 +137,23 @@ export interface Car extends CreatedUpdatedProps {
   userIDs?: string;
   carUsers?: UserCar[];
   type?: CarType;
-  converterType?: string;
+  converter?: Converter;
 }
 
+export interface Converter {
+  powerWatts: number;
```

Comment:
power should be enough. 
If you want to handle the unit, you should then add a powerUnit field.